src/pages/_id/index.vue

```diff
@@ -262,6 +240,9 @@ export default {
       const classIdSet = new Set(this.ownedNFTs.map(n => n.classId));
       return Array.from(classIdSet);
     },
+    userDisplayName() {
+      return this.userInfo?.displayName || this.wallet;
```

Comment:
This file is not TypeScript, can't use `?.`

Reply:
I think [node 14](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#browser_compatibility) now supports for `?.` syntax?

Reply:
I see, please keep optional chaining then